front/components/SearchBar/Suggestion.tsx

```diff
@@ -1,12 +1,23 @@
 import { Community } from '@/app/models/community';
+import { stringifyCommunityType } from '@/utils/utils';
 
-type CommunitySuggestionDisplayProps = Pick<Community, 'nom' | 'code_postal'>;
-
-export default function Suggestion({ nom, code_postal }: CommunitySuggestionDisplayProps) {
+type CommunitySuggestionDisplayProps = Pick<Community, 'nom' | 'code_postal' | 'type' | 'siren'>;
+export default function Suggestion({
+  nom,
+  code_postal,
+  type,
+  siren,
+}: CommunitySuggestionDisplayProps) {
+  const communityTypeText = stringifyCommunityType(type);
   return (
-    <span>
-      {nom}
-      {code_postal && <> – {code_postal}</>}
-    </span>
+    <div className='flex w-full justify-between'>
+      <span>
+        {nom} {code_postal && <>– {code_postal}</>}
```

Comment:
```suggestion
        {nom} {code_postal && <span>– {code_postal}</span>}
```
Pour rester cohérent avec le code en-dessous, mettre un span plutôt qu'un fragment

Reply:
Oui sauf que si la balise est inutile, pas la peine de surcharger le code inutilement.
Du coup, pour rendre la chose cohérente, c'est 3 lignes plus bas que je vais rendre l'ensemble cohérent en retirant la balise span inutile : 
{communityTypeText && <span> {communityTypeText}</span>} =>{communityTypeText && <> {communityTypeText}</>}